### PR TITLE
add thumbs page

### PR DESCRIPTION
**Pull request by jbesraa**
related to #64 
add thumbs components
add thumbs container
add tests to thumbs

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -10,7 +10,11 @@
 .position{
   position: fixed;
 }
-
+.center_img{
```
Comment (des-des) [inline public/style.css]: This is the same as 
`center db` in tachyons

Comment (des-des) [reply public/style.css]: If you do a search for `.center` and `.db` [here](http://tachyons.io/docs/table-of-styles/) you can see its identical

File: src/client/components/thumbs_up.js [hunk 2]
```diff
@@ -1,15 +1,19 @@
 import React, { Component } from 'react'
 
-class PositiveBtn extends Component {
+class ThumbsUpBtn extends Component {
```
Comment (des-des) [inline src/client/components/thumbs_up.js]: I think button is the wrong name. Just call it `ThumbsUp`.

File: src/client/containers/thumbs.js [hunk 3]
```diff
@@ -3,13 +3,13 @@ import { connect } from 'react-redux'
 
 import * as actions from '../actions/thumbs.js'
 
-import ThumbsUp from '../components/thumbs_up.js'
-import ThumbsDown from '../components/thumbs_down.js'
+import ThumbsUpBtn from '../components/thumbs_up.js'
+import ThumbsDownBtn from '../components/thumbs_down.js'
```
Comment (des-des) [inline src/client/containers/thumbs.js]: Again, these are not buttons

File: test/client/components/thumbs_down.js [hunk 4]
```diff
@@ -1,12 +1,19 @@
 import test from 'tape'
 import React from 'react'
 import { shallow } from 'enzyme'
-import NearMissBtn from '../../../src/client/components/thumbs_down.js'
+import ThumbsDownBtn from '../../../src/client/components/thumbs_down.js'
```
Comment (des-des) [inline test/client/components/thumbs_down.js]: name here also

File: test/client/components/thumbs_down.js [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
 
-test('<NearMissBtn />', t => {
-  const wrapper = shallow(<NearMissBtn />)
+test('<ThumbsDownBtn />', t => {
+  t.plan(2)
```
Comment (des-des) [inline test/client/components/thumbs_down.js]: :fire: `t.plan`

File: test/client/components/thumbs_up.js [hunk 6]
```diff
@@ -1,12 +1,19 @@
 import test from 'tape'
 import React from 'react'
 import { shallow } from 'enzyme'
-import PositiveBtn from '../../../src/client/components/thumbs_up.js'
+import ThumbsUpBtn from '../../../src/client/components/thumbs_up.js'
```
Comment (des-des) [inline test/client/components/thumbs_up.js]: name

File: test/client/components/thumbs_up.js [hunk 7]
```diff
@@ -1,12 +1,19 @@
...
 
-test('<PositiveBtn />', t => {
-  const wrapper = shallow(<PositiveBtn />)
+test('<ThumbsUpBtn />', t => {
+  t.plan(2)
```
Comment (des-des) [inline test/client/components/thumbs_up.js]: :fire:

**Review by des-des — CHANGES_REQUESTED** [review]

**Comment by jbesraa** [comment]
@des-des fixed

**Review by des-des — APPROVED** [review]

**Pull request merged**